index.d.ts

```diff
@@ -1,294 +1,289 @@
+export type ClickMoreEventHandlerFunc = (eventObj: IEventMoreObject) => void;
+export type ClickScheduleEventHandlerFunc = (eventObj: IEventScheduleObject) => void;
+export type TimezonesCollapseEventFunc = (timezonesCollapsed: boolean) => void;
+export type EventHandlerType = AfterRenderScheduleEventHandlerFunc |
+    BeforeCreateScheduleEventHandlerFunc |
+    BeforeDeleteScheduleEventHandlerFunc |
+    BeforeUpdateScheduleEventHandlerFunc |
+    ClickDayNameEventHandlerFunc |
+    ClickMoreEventHandlerFunc |
+    ClickScheduleEventHandlerFunc |
+    TimezonesCollapseEventFunc;
+export type CustomEventType = 'afterRenderSchedule' | 'beforeCreateSchedule' |
+    'beforeDeleteSchedule' | 'beforeUpdateSchedule' | 'clickDayname' |
+    'clickMore' | 'clickSchedule' | 'clickTimezonesCollapseBtn';
 
-    interface IEventObject {
-        schedule: ISchedule;
-        end: TZDate;
-        start: TZDate;
-        calendar?: ICalendarInfo;
-        triggerEventName?: string;
-    }
+export interface IEventObject {
+    schedule: ISchedule;
+    end: TZDate;
+    start: TZDate;
+    calendar?: ICalendarInfo;
+    triggerEventName?: string;
+}
 
-    interface IEventDateObject {
-        date: string;
-    }
+export interface IEventDateObject {
+    date: string;
+}
 
-    interface IEventMoreObject {
-        date: TZDate;
-        target: Element;
-    }
+export interface IEventMoreObject {
+    date: TZDate;
+    target: Element;
+}
 
-    interface IEventScheduleObject {
-        calendar: ICalendarInfo;
-        event: MouseEvent;
-        schedule: ISchedule;
+export interface IEventScheduleObject {
+    calendar: ICalendarInfo;
+    event: MouseEvent;
+    schedule: ISchedule;
 
-    }
-    interface IEvents {
-        'afterRenderSchedule'?: AfterRenderScheduleEventHandlerFunc;
-        'beforeCreateSchedule'?: BeforeCreateScheduleEventHandlerFunc;
-        'beforeDeleteSchedule'?: BeforeDeleteScheduleEventHandlerFunc;
-        'beforeUpdateSchedule'?: BeforeUpdateScheduleEventHandlerFunc;
-        'clickDayname'?: ClickDayNameEventHandlerFunc;
-        'clickMore'?: ClickMoreEventHandlerFunc;
-        'clickSchedule'?: ClickScheduleEventHandlerFunc;
-        'clickTimezonesCollapseBtn'?: TimezonesCollapseEventFunc;
-    }
+}
+export interface IEvents {
+    'afterRenderSchedule'?: AfterRenderScheduleEventHandlerFunc;
+    'beforeCreateSchedule'?: BeforeCreateScheduleEventHandlerFunc;
+    'beforeDeleteSchedule'?: BeforeDeleteScheduleEventHandlerFunc;
+    'beforeUpdateSchedule'?: BeforeUpdateScheduleEventHandlerFunc;
+    'clickDayname'?: ClickDayNameEventHandlerFunc;
+    'clickMore'?: ClickMoreEventHandlerFunc;
+    'clickSchedule'?: ClickScheduleEventHandlerFunc;
+    'clickTimezonesCollapseBtn'?: TimezonesCollapseEventFunc;
+}
 
-    class TZDate {
-        public getTime(): number;
-        public toDate(): Date;
-        public toUTCString(): Date;
-    }
+interface TZDate {
```

Comment:
Not entirely sure about this change. I suppose it could be exported as a class, but I believe the export is not `TZDate`: https://github.com/nhn/tui.calendar/blob/master/src/js/common/timezone.js#L252?

Reply:
Cool, thanks.